src/uct/ib/rc/base/rc_iface.c

```diff
@@ -832,12 +855,18 @@ ucs_status_t uct_rc_iface_qp_connect(uct_rc_iface_t *iface, struct ibv_qp *qp,
                                      struct ibv_ah_attr *ah_attr,
                                      enum ibv_mtu path_mtu)
 {
+    uct_ib_device_t *dev = uct_ib_iface_device(&iface->super);
     struct ibv_qp_attr qp_attr;
     long qp_attr_mask;
     int ret;
 
     ucs_assert(path_mtu != 0);
 
+    if (uct_ib_set_ece(dev, qp, iface->config.ece) != UCS_OK) {
+        ucs_error("error set qp ece");
```

Comment:
1. move this error print inside uct_ib_set_ece
2.
```
status = uct_ib_set_ece(dev, qp, iface->config.ece)
if (status != UCS_OK) {
   ...
}
```
in all places

Reply:
Done